SPECS/aspell/aspell.spec

```
@@ -1,25 +1,27 @@
-Summary: Spell checker
-Name: aspell
-Version: 0.60.8
-Release: 5%{?dist}
+Summary:        Spell checker
+Name:           aspell
+Version:        0.60.8
+Release:        6%{?dist}
 # LGPLv2+ .. common/gettext.h
 # LGPLv2  .. modules/speller/default/phonet.hpp,
 #            modules/speller/default/phonet.cpp,
 #            modules/speller/default/affix.cpp
 # GPLv2+  .. ltmain.sh, misc/po-filter.c
 # BSD     .. myspell/munch.c
-License: LGPLv2+ and LGPLv2 and GPLv2+ and BSD
+License:        LGPLv2+ AND LGPLv2 AND GPLv2+ AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL: http://aspell.net/
-Source: ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
-
-Patch0: aspell-0.60.7-fileconflict.patch
-Patch1: aspell-0.60.7-pspell_conf.patch
-Patch2: aspell-0.60.7-mp.patch
-
-BuildRequires: gcc-c++
-BuildRequires: chrpath, gettext, ncurses-devel, pkgconfig, perl-interpreter
+URL:            http://aspell.net/
+Source:         ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
```

Whenever possible, we try to use macros in source URL links. Not a big difference here, since the version is the most important part (usually makes updates easier) but for the sake of consistency:

```suggestion
Source:         ftp://ftp.gnu.org/gnu/aspell/%{name}-%{version}.tar.gz
```

---

Oh, okay. I'll update on the wiki then, since it says the opposite.